python/clx/analytics/cybert.py

```diff
@@ -69,7 +69,7 @@ def load_model(
         """
         with open(config_filepath) as f:
             config = json.load(f)
-        self._label_map = {int(k): v for k, v in config['id2label'].items()}
+        self._label_map = {int(k): v for k, v in config["id2label"].items()}
```

Comment:
This label map is typically small, correct? I'm just wondering about the loop here. If it's small though, no need to optimize now.

Reply:
Yes it is small cc @raykallen

Reply:
Then I see no reason to optimize now. Leave as-is.